packages/playwright-core/bin/reinstall_chrome_beta_linux.sh

```diff
@@ -7,6 +7,17 @@ if [[ $(arch) == "aarch64" ]]; then
   exit 1
 fi
 
+if [[ ! -f "/etc/os-release" ]]; then
+  echo "ERROR: cannot install on unknown linux distribution"
```

Comment:
maybe: `ERROR: cannot install on unknown linux distribution (/etc/os-release is missing)` ?

Reply:
done